space-plugins/nuxt-base/server/middleware/02.auth.global.ts

```diff
@@ -26,11 +26,16 @@ export default defineEventHandler(async (event) => {
 	}
 
 	event.context.appSession = appSession;
+
+	const afterAuth = appConfig.auth.middleware?.afterAuth;
+	if (typeof afterAuth === 'function') {
+		return await afterAuth({ event, appSession });
```

Comment:
This function is called only when the authentication succeeds, so I was wondering if we could name it more specifically, what do you think?

`afterAuthSucceeds`? 🤔